hw_7.py

```diff
@@ -0,0 +1,209 @@
+import json
+import os
+
+"""Задача-1
+У вас есть список(list) IP адрессов. Вам необходимо создать
+класс который будет иметь методы:
+1) Получить список IP адресов
+2) Получить список IP адресов в развернутом виде
+(10.11.12.13 -> 13.12.11.10)
+3) Получить список IP адресов без первых октетов
+(10.11.12.13 -> 11.12.13)
+4) Получить список последних октетов IP адресов
+(10.11.12.13 -> 13)
+"""
+
+
+class IPShow(object):
+    """Class to show different varieties of Ip list."""
+    def __init__(self):
+        self.__IP_list = ['135.146.235.135', '38.100.72.185', '127.27.201.98', '236.144.28.224', '57.80.195.145',
+                          '152.212.218.100', '219.20.3.53', '79.112.140.199', '33.98.75.82', '165.230.55.81',
+                          '236.78.64.193', '255.113.161.134', '116.70.239.18', '254.110.243.118', '226.241.61.154']
+
+    @property
+    def ip_list_getting(self):
+        """Method to get private Ip list."""
+        return self.__IP_list
+
+    @property
+    def reversed_ip_list(self):
+        """Method to get Ip list in reversed view."""
+        new_list = []
+        for elem in self.__IP_list:
+            new_list.append('.'.join(elem.split('.')[::-1]))
+        return new_list
+
+    @property
+    def ip_list_without_first_octets(self):
+        """Method to get IP list without first octets."""
+        new_list = []
+        for elem in self.__IP_list:
+            new_list.append('.'.join(elem.split('.')[1:]))
+        return new_list
+
+    @property
+    def ip_list_last_octets(self):
```

Comment:
all names of method should answer on question "What to do ?"  e.g. `get_ip_last_octet()`